assembl/static2/css/components/posts.scss

```diff
@@ -692,9 +736,14 @@ $deleted-post-border: 1px solid $light-grey !default;
 @media screen and (max-width: $screen-md-min) {
   .posts {
     .custom-overlay {
+      float: right;
```

Comment:
Shouldn't this be calculated by the parent view in react, based on horizontal space/position instead of being fixed?

Reply:
not in this case: here, we fixed how the bin is displayed on a mobile